tensor/stats/stats/stats.go

```diff
@@ -4,70 +4,133 @@
+	tensor.AddFunc(SemPop.FuncName(), SemPopFunc, 1)
+	tensor.AddFunc(Median.FuncName(), MedianFunc, 1)
+	tensor.AddFunc(Q1.FuncName(), Q1Func, 1)
+	tensor.AddFunc(Q3.FuncName(), Q3Func, 1)
+}
+
 // Stats is a list of different standard aggregation functions, which can be used
 // to choose an aggregation function
 type Stats int32 //enums:enum
 
 const (
-	// count of number of elements
+	// count of number of elements.
 	Count Stats = iota
 
-	// sum of elements
+	// sum of elements.
 	Sum
 
-	// product of elements
+	// sum of absolute-value-of elements (= L1Norm).
+	SumAbs
+
+	// L1 Norm: sum of absolute values (= SumAbs).
+	L1Norm
+
+	// product of elements.
 	Prod
 
-	// minimum value
+	// minimum value.
 	Min
 
-	// max maximum value
+	// maximum value.
 	Max
 
-	// minimum absolute value
+	// minimum of absolute values.
 	MinAbs
 
-	// maximum absolute value
+	// maximum of absolute values.
 	MaxAbs
 
-	// mean mean value
+	// mean value = sum / count.
 	Mean
 
-	// sample variance (squared diffs from mean, divided by n-1)
+	// sample variance (squared deviations from mean, divided by n-1).
 	Var
 
-	// sample standard deviation (sqrt of Var)
+	// sample standard deviation (sqrt of Var).
 	Std
 
-	// sample standard error of the mean (Std divided by sqrt(n))
+	// sample standard error of the mean (Std divided by sqrt(n)).
 	Sem
 
-	// L1 Norm: sum of absolute values
-	L1Norm
-
-	// sum of squared values
+	// sum of squared values.
 	SumSq
 
-	// L2 Norm: square-root of sum-of-squares
+	// L2 Norm: square-root of sum-of-squares.
 	L2Norm
 
-	// population variance (squared diffs from mean, divided by n)
+	// population variance (squared diffs from mean, divided by n).
 	VarPop
 
-	// population standard deviation (sqrt of VarPop)
+	// population standard deviation (sqrt of VarPop).
 	StdPop
 
-	// population standard error of the mean (StdPop divided by sqrt(n))
+	// population standard error of the mean (StdPop divided by sqrt(n)).
 	SemPop
 
-	// middle value in sorted ordering
+	// middle value in sorted ordering.
 	Median
 
-	// Q1 first quartile = 25%ile value = .25 quantile value
+	// Q1 first quartile = 25%ile value = .25 quantile value.
 	Q1
 
-	// Q3 third quartile = 75%ile value = .75 quantile value
+	// Q3 third quartile = 75%ile value = .75 quantile value.
 	Q3
 )
+
+// FuncName returns the package-qualified function name to use
+// in tensor.Call to call this function.
+func (s Stats) FuncName() string {
+	return "stats." + s.String()
+}
+
+// StripPackage removes any package name from given string,
+// used for naming based on FuncName() which could be custom
+// or have a package prefix.
+func StripPackage(name string) string {
+	spl := strings.Split(name, ".")
+	if len(spl) > 1 {
+		return spl[len(spl)-1]
+	}
+	return name
+}
+
+// Stat calls a standard Stats enum function on given tensors.
+// Output results are in the out tensor.
+func Stat(stat Stats, in, out *tensor.Indexed) {
```

Comment:
Again, we are in `stats`. What about `stats.Call` as a clearer name?

Reply:
again, prefer specific names and package might have other functions, e.g., I just moved norm functions in there b/c they fit better.